packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,931 @@
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+The [CAPTCHA](#captcha-object)
+
+#### Response body
+
+Returns updated [CAPTCHA Response](#captcha-response-object).
+
+#### Use example
+
+This example fully updates a CAPTCHA with `PUT` method:
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "myReCaptcha",
+  "siteKey": "copy_your_site_key",
+  "secretKey": "copy_your_secret_key",
+  "type": "RECAPTCHA_V2"
+}' "https://${yourOktaDomain}/api/v1/captchas/{id}"
+```
+
+##### Response
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "id": "cap18c2Ey3iR9BLDb0g4",
+  "name": "myReCaptcha",
+  "siteKey": "6LeVzVIaAAAAAHCbhZ-uxTihGl7iuufxEF_dMH-x",
+  "type": "RECAPTCHA_V2",
+  "_links": {
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+
+This example partially updates a CAPTCHA with `POST` method:
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "siteKey": "new_site_key",
+  "secretKey": "new_secret_key"
+}' "https://${yourOktaDomain}/api/v1/captchas/{id}"
+```
+
+##### Response
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "id": "cap18c2Ey3iR9BLDb0g4",
+  "name": "myReCaptcha",
+  "siteKey": "new_site_key",
+  "type": "RECAPTCHA_V2",
+  "_links": {
+    "self": {
+      "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Delete CAPTCHA
+
+<ApiOperation method="delete" url="/api/v1/captchas/{id}" />
+
+Permanently deletes a CAPTCHA.
+
+> Important: If the CAPTCHA is associated with org-wide CAPTCHA settings, it cannot be removed before unassociated with [org-wide CAPTCHA settings](#org-wide-captcha-settings-object).
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `id `       | URL        | Required. ID of CAPTCHA. String. |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+N/A
+
+#### Usage example
+
+The following request would delete a CAPTCHA with an `id`:
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/captcha/{id}"
+```
+
+##### Response
+
+```http
+HTTP/1.1 204 No Content
+Content-Type: application/json
+```
+#### Error example
+The following request would return error when the CAPTCHA associated with [org-wide CAPTCHA settings](#org-wide-captcha-settings-object).
```

Comment:
```suggestion
The following request would return an error because the CAPTCHA is associated with [org-wide CAPTCHA settings](#org-wide-captcha-settings-object).
```